docs/client.html

```diff
@@ -106,6 +106,7 @@
 <li class="toctree-l1"><a class="reference internal" href="datasets.html">Datasets</a></li>
 <li class="toctree-l1"><a class="reference internal" href="model_groups.html">Model Groups</a></li>
 <li class="toctree-l1"><a class="reference internal" href="jobs.html">Jobs and Job Status</a></li>
+<li class="toctree-l1"><a class="reference internal" href="review-submissions.html">Review Submissions</a></li>
```

Comment:
do we want to switch all of these as well?

Reply:
Hmmm- not sure why Sphinx didn't automatically update all of the references and delete that html all together, will investigate

Reply:
these have all been updated